### DISCUSSION TITLE
Kostenfelder Wartungen/Prüfungen

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#19] [Shipped - Live on production]
Kosten direkt im Event hinterlegen. 
- Was hat der Vorgang gekostet?
-> Übersicht auf der Inventarmaske über die Gesamtkosten für das Gerät

**Comment by moritzvolkmar** [comment]
- Brutto und/oder Netto Felder?
- Einzelne Felder für Material, Service & Personal? oder ein gemeinsames Feld für alles?

**Comment by YvesR** [comment]
- Netto, ein Preis pro Vorgang (issue)
- Am Inventar die Summe der Vorgänge pro Typ (`malfunction maintenance security_message occurrence device_retired`)
- Total über alle Kosten

Inventar erweitern:
```

 {
   issue costs: {
     total: 500.00,
     malfunction: 300.00,
     maintenance: 200.00,
     security_message: 0.00,
     occurrence: 0.00,
     device_retired: 0.00
   }
 }

```
